packages/client/src/clients/feed/interfaces.ts

```diff
@@ -22,6 +22,10 @@ export interface FeedClientOptions {
   trigger_data?: GenericData;
   // Optionally enable cross browser feed updates for this feed
   __experimentalCrossBrowserUpdates?: boolean;
+  // Optionally automatically manage socket connections on changes to tab visibility
+  // Defaults to false. If true, defaults to disconnecting after 2000ms when the tab is hidden
+  // Accepts a number as a custom delay duration in milliseconds
+  auto_manage_socket_connection?: boolean | number;
```

Comment:
🤔 I wonder if this should be 2 options, one for enabling the behavior and one for setting the disconnect window.

Reply:
Yeah, was debating that. Having one option could be a little confusing and requires more explanation. Having two options is more verbose and requires some extra logic to handle both cases (not a problem, just something to consider if e.g. a duration is set but the boolean flag is off)

Reply:
Going to refactor to have two options. I think it's easier to understand